lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastReceiver.cs

```diff
@@ -79,6 +76,17 @@ public sealed class BroadcastReceiver<T> : IBroadcastReceiver<T>
         public string OperatorName { get; private set; }
 
         /// <summary>
+        /// Ensure all parent and children nodes in the topology are registered with teh Name Service.
+        /// </summary>
+        public void WaitForRegistration()
```

Comment:
We could have an `internal` interface that extends the `public` one and only adds this one `internal` method, right? That way, we don't have to advertise this method into the IntelliSense of our users.